eav/queryset.py

```diff
@@ -282,3 +282,78 @@ def get(self, *args, **kwargs):
         the ``Manager`` get method.
         """
         return super(EavQuerySet, self).get(*args, **kwargs)
+
+    def order_by(self, *fields):
+        # Django only allows to order querysets by direct fields and
+        # foreign-key chains. In order to bypass this behaviour and order
+        # by EAV attributes, it is required to construct custom order-by
+        # clause manually using Django's conditional expressions.
+        # This will be slow, of course.
+        order_clauses = []
+        query_clause = self
+
+        for term in [t.split('__') for t in fields]:
+            # Continue only for EAV attributes.
+            if len(term) == 2 and term[0] == 'eav':
```

Comment:
This means that you do not contain the whole `eav` space for yourself, e.g. `eav__something__something` raises other possible errors than `eav__something`. Consider possibly claiming the whole space for integrity?

Reply:
I'm raising `NotImplementedError` when ordering by foreign-key chains is attempted (e.g. `eav__something__something`).

Reply:
If we are splitting hairs, then: https://docs.djangoproject.com/en/2.1/_modules/django/db/utils/#NotSupportedError is better.